hathor/cli/util.py

```diff
@@ -230,7 +230,12 @@ def setup_logging(
 
     def kwargs_formatter(_, __, event_dict):
         if event_dict and event_dict.get('event') and isinstance(event_dict['event'], str):
-            event_dict['event'] = event_dict['event'].format(**event_dict)
+            try:
+                event_dict['event'] = event_dict['event'].format(**event_dict)
+            except KeyError:
+                # The event string may contain '{}'s that are not used for formatting, resulting in a KeyError in the
+                # event_dict. In this case, we don't format it.
+                pass
         return event_dict
```

Comment:
I would prefer to simply remove this `kwargs_formatter()` function instead, then we would stop supporting log calls like this:

```python
log.info('logging {something}', something=123)
```

We can always use f-strings or `str.format()` directly instead. But that would require searching through the project to convert those calls, and it may not be trivial to find all occurrences.

Reply:
From what I remember, when I initially wrote this function I tested removing support for the `kwargs_formatter` but it broke log calls by twisted itself, otherwise those logs would be literally displayed as `logging {something}` with `something=123` shown separately.

Reply:
Ahh ok. It's weird that twisted depends on us providing this function, there's probably another way to do it, but I'm fine keeping it for now, at least the problem was fixed.

Reply:
It's not that twisted depends on us providing this functions, it's more that twisted's logging system implements this and if we are to replace it we have to keep this behavior.

Reply:
@jansegre maybe we should copy their implementation, then?